client/elements/sc-page-dictionary.html

```diff
@@ -193,90 +146,70 @@ <h3>Similar Spelling</h3>
+      class SCPageDictionary extends ReduxMixin(Polymer.Element) {
+          static get is() {
+              return 'sc-page-dictionary';
+          }
+
+          static get properties() {
+              return {
+                  dictData: {
+                      type: Object
+                  },
+                  loadingDict: {
+                      type: Boolean,
+                      value: true
+                  },
+                  dictionaryParams: {
+                      type: Object,
+                      statePath: 'dictionaryParams'
+                  },
+                  dictionaryTitles: {
+                      type: Object,
+                      value: {
+                          'ncped': 'New Concise Pali English Dictionary',
+                          'cped': 'Concise Pali English Dictionary',
+                          'dhammika': 'Nature and the Environment in Early Buddhism by S. Dhammika',
+                          'dppn': 'Pali Proper Names',
+                          'pts': 'PTS Pali English Dictionary'
+                      }
                   }
-              },
-              inputLanguage: String,
-
-              // checks if item is found in any of the 5 dictionaries
-              cpedItem: {
-                  type: String,
-                  computed: '_checkDictionaryEntry(cpedResult, dictionaryItem)'
-              },
-              ncpedItem: {
-                  type: String,
-                  computed: '_checkDictionaryEntry(ncpedResult, dictionaryItem)'
-              },
-              dhammikaItem: {
-                  type: String,
-                  computed: '_checkDictionaryEntry(dhammikaResult, dictionaryItem)'
-              },
-              ptsItem: {
-                  type: String,
-                  computed: '_checkDictionaryEntry(ptsResult, dictionaryItem)'
-              },
-              dppnItem: {
-                  type: String,
-                  computed: '_checkDictionaryEntry(dppnResult, dictionaryItem)'
-              }
-          },
-          // checks if item is found in this specific dictionary and returns the result.
-          _checkDictionaryEntry: function(dictResult, dictionaryItem) {
-              if (dictResult[dictionaryItem.toLowerCase()]) {
-                  return dictResult[dictionaryItem.toLowerCase()];
-              } else if (dictResult[dictionaryItem.charAt(0).toUpperCase() + dictionaryItem.slice(1)]) {
-                  return dictResult[dictionaryItem.charAt(0).toUpperCase() + dictionaryItem.slice(1)];
               }
-          },
+          }
+
+          _computeUrl() {
+              console.log("compute URL");
+              return `${this.$.meta.byKey('API_ROOT')}/dictionary_full`;
+          }
+
+          _didRespond() {
+              console.log("Ajax has responded");
+              console.log(this.dictData);
+          }
+
+          _generateRequest() {
+              this.shadowRoot.querySelector('#ajax').generateRequest();
```

Comment:
You can use `this.$.ajax` instead of querySelector
https://www.polymer-project.org/2.0/docs/devguide/dom-template   Static node map